test/browser/spec/ol/interaction/snap.test.js

```diff
@@ -71,12 +71,22 @@ describe('ol.interaction.Snap', function () {
         coordinate: [0, 0],
         map: map,
       };
+      snapInteraction.on('snap', function (snapEvent) {
+        expect(snapEvent.type).to.be('snap');
+        expect(snapEvent.vertex).to.be(event.coordinate);
+        expect(snapEvent.vertexPixel).to.be(event.pixel);
+        expect(snapEvent.feature).to.eql(point);
+
+        // check that the coordinate is in XY and not XYZ
+        expect(event.coordinate).to.eql([0, 0]);
```

Comment:
* Moved event.coordinate check inside eventHandler check
* keeping it in the same unit test for ease of maintenance
* reuse `event` input data